setup.py

```diff
@@ -68,7 +68,7 @@ def read(fname):
 
     packages=find_packages(),
     package_data={
-        "smart_open.tests": ["test_data/*gz"],
```

Comment:
Wasn't me ;-( - where does it come from ?

Reply:
No big deal. You branched your work from the master branch, but you're trying to merge it to the develop branch. This is why your PR includes work from other people. I rebased your PR off develop.

In the future, please branch off develop.